### PR TITLE
Meenu:/ fix - calculator page bug fix

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

Example calculation box is aligned same with other containers. 

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx [hunk 1]
```diff
@@ -229,14 +226,11 @@ export const StopLossLevelUp = () => {
...
+                                <PointerContainerMobile mw="0" ml="-20px">
                                     <PointerDotMobile />
-                                    <PointerStickMobile />
-                                    <PointerTextMobile>
-                                        <Localize translate_text="Stop loss" />
-                                    </PointerTextMobile>
+                                    <PointerStickMobile height="18px" />
                                     <PointerTextMobile>
-                                        <Localize translate_text="level" />
+                                        <Localize translate_text="Stop loss level" />
```
Comment (prince-deriv) [inline src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx:233]: Do we need translation for this?

Comment (meenakshi-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx:233]: @prince-deriv I think Yes, that's why they wanted to convert from image to code

Comment (prince-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx:233]: @meenakshi-deriv do you know if they need the translation right away or we can have it later?

Comment (meenakshi-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx:240]: @habib-deriv Can you please advice

**Review by habib-deriv — APPROVED** [review]

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by habib-deriv — CHANGES_REQUESTED** [review]
LGTM

**Pull request merged**